src/System.Runtime.WindowsRuntime/src/System/Runtime/InteropServices/WindowsRuntime/MarshalingHelpers.cs

```diff
@@ -89,12 +111,43 @@ internal static IntPtr ConvertToNative(NotifyCollectionChangedEventArgs managedA
             if (managedArgs == null)
                 return IntPtr.Zero;
 
-            return System.StubHelpers.EventArgsMarshaler.CreateNativeNCCEventArgsInstance(
-                        (int)managedArgs.Action,
-                        managedArgs.NewItems,
-                        managedArgs.OldItems,
-                        managedArgs.NewStartingIndex,
-                        managedArgs.OldStartingIndex);
+            IntPtr newItemsIP = IntPtr.Zero;
+            IntPtr oldItemsIP = IntPtr.Zero;
+
+            RuntimeHelpers.PrepareConstrainedRegions();
```

Comment:
Nit: `PrepareConstrainedRegions` does nothing in CoreCLR. We do not bother with these annotations anymore.

Reply:
Removed